fares-data-build-tool/create-data
# feat: splitting up matching json for each product

# Description

-   To create a distinct matching JSON for every product the user has created, and put them into the products table / bucket.

# Testing instructions

-   Run locally and make sure the correct amount of matching JSON gets created (there should also be the FULL matching JSON created, but that should not be added to the products table).

# Type of change
